{"uuid": "5f474240-f86a-40d0-b55e-d92c84115596", "children": ["c7d56e52-ac8a-44d8-8009-ac322516ee51"], "befores": [{"name": "_unittest_setUpClass_fixture_TestGymManagement", "status": "broken", "statusDetails": {"message": "AttributeError: type object 'TestManagementUserAuth' has no attribute 'access_token'\n", "trace": "  File \"C:\\Users\\jothi\\Documents\\api-testing-repo\\.venv\\lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n  File \"C:\\Users\\jothi\\Documents\\api-testing-repo\\.venv\\lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"C:\\Users\\jothi\\Documents\\api-testing-repo\\.venv\\lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"C:\\Users\\jothi\\Documents\\api-testing-repo\\.venv\\lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\jothi\\Documents\\api-testing-repo\\.venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\jothi\\Documents\\api-testing-repo\\.venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 922, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\jothi\\Documents\\api-testing-repo\\.venv\\lib\\site-packages\\_pytest\\unittest.py\", line 148, in unittest_setup_class_fixture\n    setup()\n  File \"C:\\Users\\jothi\\Documents\\api-testing-repo\\tests\\test_gym_management.py\", line 29, in setUpClass\n    if not TestManagementUserAuth.access_token:\n"}, "start": 1754319822631, "stop": 1754319822633}], "start": 1754319822631, "stop": 1754319822976}